test/TypeCheck/Positive.hs

```diff
@@ -87,6 +87,10 @@ tests =
       "Implicit arguments"
       "MicroJuvix"
       "Implicit.juvix",
+    PosTest
+      "Pattern match a hole type"
+      "265"
```

Comment:
Better if you name the folder as "issue265". The number alone doesn't mean much.

Reply:
0k